api/CSS.json

```diff
@@ -1635,10 +1635,10 @@
               "version_added": "12.1"
             },
             "safari": {
-              "version_added": "9"
+              "version_added": false
```

Comment:
This change doesn't make a lot of sense to me. It doesn't seem to be resolving an inconsistency and it's introducing data that's just not correct (I can confirm that `CSS.supports` works in Safari 12, having just tested it). Is there more to the story here?

I don't have an iOS device to test on, but I think there may be a similar issue with the change after this one too

Reply:
It looks like api.CSS was wrongly stated as unsupported in Safari and Safari iOS during the initial commit (or rather, the migration from the MDN tables, I assume).  This led to Safari iOS being set to false down below, and I guess I accidentally changed this line as well.  Reverting both changes!